### ISSUE TITLE
Add support for Drag and Drop file loading

### ISSUE THREAD

**Issue by Mhowser** [niftools/blender_niftools_addon#422]
**Is your feature request related to a problem? Please describe.**
I have a ton of NIF files that need to be imported into blender, and it is quite slow manually importing each one.

**Describe the solution you'd like**
Simply click and drop the file into blender

**Describe alternatives you've considered**
N/A

**Additional context**
This currently does not work.

**Comment by HENDRIX-ZT2** [comment]
We can't implement this, blender devs have to support drag and drop first.

**Comment by HENDRIX-ZT2** [comment]
The only thing we can do is allow for multiple nifs to be selected on import, like we're doing for kf.

**Comment by neomonkeus** [comment]
Yeah I attempting to find out more if Blender supported associating custom file extensions with addons &/or drag & drop functionality, but they aren't super responsive on blender.chat. Not sure it that is separate to the old IRC chatrooms they have, but I do see lots of devs I recognise, so a bit surprised. There are few items in their backlog, but no real tangible work to enable us to support this request currentlt.
